daemon/sdnotify.go

```diff
@@ -2,30 +2,33 @@
 package daemon
 
 import (
-	"errors"
 	"net"
 	"os"
 )
 
-var SdNotifyNoSocket = errors.New("No socket")
-
 // SdNotify sends a message to the init daemon. It is common to ignore the error.
```

Comment:
docstring should be updated to document the three possible cases:
 * (true, nil) - notification supported, data has been sent
 * (false, nil) - notification not supported
 * (false, err) - notification supported, but failure happened